### ISSUE TITLE
Unfix #40

### ISSUE THREAD

**Issue by leifoolsen** [leifoolsen/mdl-ext#43]
The focus outline removed in #40 should be reset to it's state before fix. The problem was that the client forgot to call `componentHandler.upgradeElement(element, 'MaterialExtSelectfield')` after inserting a selectield. Showing the focus outline provides a fallback if the `upgradeElement` method is not called.
